src/LiveDevelopment/BrowserScripts/LiveDevProtocolRemote.js

```diff
@@ -382,12 +382,146 @@
     }
 
 
+    /**
+    * Gets the word at the clicked position along with additional information
+    * @param {Element} element - The element that was clicked
+    * @param {MouseEvent} event - The click event
+    * @return {Object|null} - Object containing the word and additional info, or null if not found
+    */
+    function getClickedWord(element, event) {
+
+        // Try to find the clicked position within the element
+        const range = document.caretRangeFromPoint(event.clientX, event.clientY);
```

Comment:
This api doesn't work in Firefox, so maybe we can fallback to [caretPositionFromPoint](https://developer.mozilla.org/en-US/docs/Web/API/Document/caretPositionFromPoint) in Firefox(caretPositionFromPoint doesnt work in safari), or disable the feature in Firefox.